src/popup/pages/AccountDetail/styles.ts

```diff
@@ -116,6 +116,11 @@ const styles: StyleRulesCallback = (theme: Theme) => ({
     minHeight: 0,
     padding: `0px ${theme.padding.unit}`,
   },
+  amountInfo: {
+    display: 'flex',
+    width: '100px',
```

Comment:
lets not set a max width on the amount container. this also affects the `Transaction row` amount as well and we show up to 8 decimal places which is greater than `100px`.

you should be able to set the width/height on the icon and it should also be in the same position.